src/Todo.test.tsx

```diff
@@ -1,21 +1,32 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import {fireEvent, render, screen} from '@testing-library/react';
 import Todo from './Todo';
+import {mockAddTodo, mockFetchTodoList} from "./mocks/mock";
 
-test('Has a submit button', () => {
-  render(<Todo/>);
-  const submitButton = screen.getByText('Add');
-  expect(submitButton).toBeInTheDocument();
+test('Has a submit button',   () => {
+    render(<Todo fetchList={mockFetchTodoList} addTodo={mockAddTodo}/>)
```

Comment:
Your formatting isn't matching - you've got some with 2 spaces and some with 4.  Make sure it's all consistent.

Reply:
Will update.